### ISSUE TITLE
Flickering on updates in tmux

### ISSUE THREAD

**Issue by ibuda** [achannarasappa/ticker#60]
Hi, I was wondering if I am the only one experiencing flickering on updates of the tickers list.
I've just installed the package, and for demo purpose added AAPL,NET,TSLA tickers.
I use tmux, could be this the source of the problem?
I tested it on simple terminal, it works ok.

My os is ubuntu 20.04.

**Comment by 097115** [comment]
Yeah, I'm also experiencing that. You have a split window by any chance? Or just a relatively small one may be?

**Comment by ibuda** [comment]
@097115  I do have my window split, the pane is not small though, as it does not run if the pane is smaller than a certain threshold (do not remember which one). But no matter how big I make the pane, it still flickers.
If I open a new window though and run the ticker there, the flickering stops.

**Comment by 097115** [comment]
> If I open a new window though and run the ticker there, the flickering stops.

A new tmux non-splitted window? Or a new terminal window without tmux? 

Anyway, can't really help much except for confirming this probably tmux-related flickering in some situations (doesn't seem to flicker in GNU Screen, by the way)

**Comment by achannarasappa** [comment]
This should be resolved in the latest version. Let me know if the issue persists and I'll reopen the issue

**Comment by ezeklusman** [comment]
Hi, using it on OSX I get the flickering on updates.
Wasn't happening before, it started after formatting and reinstalling ticker.
Any data I can provide to help?

**Comment by ezeklusman** [comment]
@achannarasappa can this issue be reopened?

**Comment by uxjulia** [comment]
I'm also experiencing flickering on updates. I'm using iTerm2 on OSX
